lib/engines/hcktest/hcktest.rb

```diff
@@ -226,29 +226,44 @@ def tag
       end
     end
 
-    def manual_run
+    def dump_only_run
       ResourceScope.open do |scope|
         run_studio(scope, { dump_only: true })
         run_clients(scope, { dump_only: true })
       end
     end
 
+    def wait_manual
+      return unless @project.options.test.manual
+
+      @project.logger.info('AutoHCK started in manual mode. Waiting for manual exit.')
+      loop do
+        @project.logger.info("Type 'exit' and press ENTER to exit AutoHCK")
+
+        break if $stdin.gets&.chomp == 'exit'
+      end
```

Comment:
What about invoking irb? c.f., https://docs.ruby-lang.org/en/3.3/Binding.html#method-i-irb

Reply:
1. RuboCop does not like this
2. I don't like this allows users to access a lot of internal states.

Reply:
1. RuboCop is confused and thinks you left code for debugging. Just ignore it.
2. You can create a `Binding` at a different scope to limit the state visibility. It is also possible to use `Ractor` or even create another Ruby process and perform IPC for strong isolation, but I don't think it's worthwhile.